examples/echo.rs

```diff
@@ -34,13 +28,15 @@ fn echo(req: Request<Body>) -> BoxFut {
         // Convert to uppercase before sending back to client.
         (&Method::POST, "/echo/uppercase") => {
             let mapping = req.into_body().map(|chunk| {
-                chunk
+                let chunk= chunk.unwrap();
+                let x = chunk
                     .iter()
                     .map(|byte| byte.to_ascii_uppercase())
-                    .collect::<Vec<u8>>()
-            });
+                    .collect::<Vec<_>>();
+                x
+            }).collect::<Vec<_>>().await;
```

Comment:
Hm, this `await` changes the behavior a little. Before, a response would be sent immediately, and the body would be streamed as it was received. By waiting on the body to complete here, a response is sent much later. We should probably keep to the `wrap_stream` approach.

Reply:
Yeah, wondering how to do that, it seems that `wrap_stream` is not available anymore, it is commented out right now. Do you have any tips on what to do about it?

Reply:
Oh, woops! If you grab master again, I've just re-enabled `wrap_stream`!

Reply:
Using `wrap_stream` seems working now. Just this `concat2` issue below now...